packages/cli/src/commands/__tests__/test.test.js

```diff
@@ -19,6 +20,13 @@ jest.mock('@redwoodjs/structure', () => {
   }
 })
 
+jest.mock('fs', () => {
+  return {
+    ...jest.requireActual('fs'),
+    existsSync: () => true,
+  }
+})
+
```

Comment:
@josemasar I do think this is an ok choice here. Because the new code is a functional catch for misconfiguration, it's not necessary to add a test to avoid regression (any conceivable changes to template config will be test enough). Effectively all that needs to be done is to make sure the tests can run without the new code interferring.

If there were previous uses of existsSync, we'd want to be more precise. Theoretically someone could add code that uses existsSync and the tests result in a false positive. But I'll just add a comment here to state why this is added in the first place.